net/tor/src/opnsense/service/templates/OPNsense/Tor/tor

```diff
@@ -0,0 +1 @@
+tor_enable="{% if helpers.exists('OPNsense.tor.general.enabled') and OPNsense.tor.general.enabled == '1' %}YES{% else %}NO{% endif %}"
```

Comment:
muli-line please. this is missing tor_opnsense_bootup_run=".../setup.sh" in YES case anyway